editor/src/components/inspector/fill-hug-fixed-control.tsx

```diff
@@ -279,7 +284,7 @@ export const FillHugFixedControl = React.memo<FillHugFixedControlProps>((props)
 
   const controlStylesRef = React.useRef(getControlStyles('simple'))
```

Comment:
actually, this is all sorts of wrong!

Reply:
If I read this correctly, and this is just a constant simple control style, then we should make it a simple `const` at the file root

Reply:
fixed in https://github.com/concrete-utopia/utopia/pull/3270/commits/6ed5ddb5b563de88bbacd03af1996e5d02ceed4f